models/wishlist.py

```diff
@@ -0,0 +1,38 @@
+from init_db import db, ma
+
+
+class Like(db.Model):
+    __tablename__ = 'wish_list_likes'
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+    wish_item_id = db.Column(db.Integer, db.ForeignKey('wish_list_items.id'), nullable=False)
+
+    def __repr__(self):
+        return '<Like {}>'.format(self.wish_item_id, self.user_id)
+
+
+class LikeSchema(ma.ModelSchema):
```

Comment:
Let's store schemas outside of /models. You could do it in /serializers/wishlist.py for example. It's minor.